src/Compilers/CSharp/Portable/Lowering/LambdaRewriter/SynthesizedClosureMethod.cs

```diff
@@ -164,6 +164,12 @@ protected override ImmutableArray<TypeSymbol> ExtraSynthesizedRefParameters
         internal override bool IsExpressionBodied => false;
         internal MethodSymbol TopLevelMethod => _topLevelMethod;
 
+        public override ImmutableArray<CSharpAttributeData> GetAttributes()
```

Comment:
@AlekseyTs pointed out that we may want to debug assert that if you call the `base` method, it returns nothing. That ensures that we aren't dropping any attributes here.